packages/realm-web/src/User.ts

```diff
@@ -240,17 +201,37 @@ export class User implements Realm.User {
 
     public async logOut() {
         // Invalidate the refresh token
-        await this.app.baseTransport.fetch({
-            method: "DELETE",
-            path: "/auth/session",
-            headers: {
-                Authorization: `Bearer ${this._refreshToken}`,
-            },
-        });
-        // Forget the tokens
-        this._accessToken = null;
-        this._refreshToken = null;
+        if (this._refreshToken !== null) {
+            await this.app.baseTransport.fetch({
+                method: "DELETE",
+                path: "/auth/session",
+                headers: {
+                    Authorization: `Bearer ${this._refreshToken}`,
+                },
+            });
+            // Forget the tokens
+            this._accessToken = null;
+            this._refreshToken = null;
+        }
         // Update the state
         this._state = UserState.LoggedOut;
     }
+
+    /** @inheritdoc */
+    public async linkCredentials(credentials: Realm.Credentials) {
+        throw new Error("Not yet implemented");
+    }
+
+    public async refreshCustomData() {
+        await this.refreshAccessToken();
```

Comment:
So in reality, an exception is thrown, right?

Reply:
Yeah, for now :)